locale/it/site.json

```diff
@@ -10,6 +10,8 @@
     "by": "da",
     "all-downloads": "Tutti i download",
     "nightly": "Build notturne",
+    "previous": "Precedente",
```

Comment:
It's probably to better to use "Indietro/Avanti".
If the context is "page" then it's ok but "next" should be "Successiva".